index.bs

```diff
@@ -206,7 +181,7 @@ spec: RFC7240; urlPrefix: https://tools.ietf.org/html/rfc7240
   If we assume that authors do the server-side legwork (obtaining a certificate,
   configuring the server, setting up redirects), and that authors also ensure
   that both first- and third-party content is accessible at <em>the same
-  {{URL/host}} and {{URL/path}}</em> on a secure {{URL/scheme}}, then the
+  <a for=url>host</a> and <a for=url>path</a></em> on a secure <a for=url>scheme</a>, then the
```

Comment:
Nit: If we wanted to stick with shortcuts, `[=url/host=]` is shorter. I prefer that aesthetic as easier to read than `<a>...</a>`, but I don't have a strong preference.

Reply:
I think I fixed this now.